src/axiosCromwell.ts

```diff
@@ -1,10 +1,51 @@
 import axios from 'axios';
+import { useDispatch } from "react-redux";
+import { sessionExpired } from "./store/auth/actions";
 
 const axiosCromwell = axios.create({
-  baseURL: 'http://localhost:3001/',
+  baseURL: 'http://localhost:8080/',
   headers: {
     'Content-Type': 'application/json'
   }
 });
 
+axiosCromwell.interceptors.request.use(
+  config => {
+    const accessToken = localStorage.getItem('accessToken');
+    if (accessToken) {
+      config.headers["Authorization"] = accessToken;
+    }
+    return config;
+  },
+  error => {
+    Promise.reject(error);
+  }
+);
+
+axiosCromwell.interceptors.response.use(
+  response => {
+    return response;
+  },
+  error => {
+    const originalRequest = error.config;
+    let refreshToken = localStorage.getItem('refreshToken');
```

Comment:
As you're not changing `refreshToken`, you can declare it as `const` as well

Reply:
You are totally correct :+1: